src/generator/AutoRest.Go/CodeNamerGo.cs

```diff
@@ -0,0 +1,491 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+
+using AutoRest.Core;
+using AutoRest.Core.Utilities;
+using AutoRest.Core.Utilities.Collections;
+using AutoRest.Core.Model;
+using AutoRest.Go.Model;
+
+namespace AutoRest.Go
+{
+    public class CodeNamerGo : CodeNamer
+    {
+        public new static CodeNamerGo Instance
+        {
+            get
+            {
+                return (CodeNamerGo)CodeNamer.Instance;
+            }
+        }
+
+        // Refactor -> generator
+        public static readonly List<string> AutorestImports = new List<string> { "github.com/Azure/go-autorest/autorest" };
```

Comment:
>public static [](start = 8, length = 13)

Plus we're trying to get away from public static *anything* in the namer classes. They should be `public virtual` properties, so that they may be overriden in subclasses when necessary (and `Namer.Instance ` is the static accessor to the current instance.